src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
LGTM, except I think maybe AUTOBRIDGE_CALCULATE_THROTTLE_TIME should be set to 1000 - it keeps the default faster and makes the escalation more granular at the low end, if I did my math correctly:

AUTOBRIDGE_CALCULATE_THROTTLE_TIME @ 4000:

| Compute Time | Multiplier | Throttle Timeout |
| ------------- | ----------- | ---------- |
| 1000 | 1 | 4000 |
| 2000 | 2 | 8000 | 
| 4000 | 3 | 12000 |

AUTOBRIDGE_CALCULATE_THROTTLE_TIME @ 1000:

| Compute Time | Multiplier | Throttle Timeout |
| ------------- | ----------- | ---------- |
| 250 | 1 | 1000 |
| 500 | 2 | 2000 | 
| 1000 | 3 | 3000 |
| 2000 | 4 | 5000 |
| 4000 | 9 | 9000 |